tests/js/cases/system/framework.test.js

```diff
@@ -1,673 +0,0 @@
-import { assert } from 'chai'
```

Comment:
Are we not running the tests for the original framework.js anymore or are they not really that useful?

Reply:
If we're not going to be updated the original framework anymore (beyond the parent form functionality), it probably won't be necessary to test the old framework. It just doesn't change enough to justify running tests on it all the time.